clang_build/cli.py

```diff
@@ -9,11 +9,7 @@
 from pathlib import Path as _Path
 from multiprocessing import freeze_support as _freeze_support
 
-from pbr.version import VersionInfo as _VersionInfo
-_v = _VersionInfo('clang-build').semantic_version()
-__version__ = _v.release_string()
-version_info = _v.version_tuple
-
+import clang_build as _clang_build
```

Comment:
since only `__version__` is used, maybe better just `from clang_build import __version__ as _version`?

Reply:
k